src/pages/appel-a-conferences.jsx

```diff
@@ -0,0 +1,238 @@
+// vendors
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+// components
+import Button from '../components/Button';
+import Center from '../components/LayoutSections/Center';
+import SectionContainer from '../components/SectionContainer';
+import SEO from '../components/SEO';
+
+// utils
+import breakpointsRange from '../utils/breakpointsRange';
+
+// views
+
+// styles
+import breakpoints from '../styles/breakpoints';
+import { titleStyle, introStyle } from '../styles/global';
+import Stack from '../components/LayoutSections/Stack';
+import { fontFamilies, fontWeights } from '../styles/typography';
+import colors from '../styles/colors';
+
+const SpeakersTitle = styled.h1`
+  ${breakpointsRange(
+    [
+      { prop: 'marginTop', sizes: [60, 150], bases: [16, 20] },
+      { prop: 'marginBottom', sizes: [60, 80], bases: [16, 20] },
+    ],
+    breakpoints.spacings
+  )};
+`;
+
+const SpeakersIntro = styled.div`
+  margin-bottom: 0;
+
+  ${breakpointsRange(
+    [{ prop: 'marginTop', sizes: [16, 16] }],
+    breakpoints.spacings,
+    { bases: [16, 20] }
+  )};
+`;
+
+const VideoContainer = styled.div`
+  position: relative;
+
+  padding-top: 56.25%;
+`;
+
+const Video = styled.iframe`
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 2;
+
+  width: 100%;
+  height: 100%;
+`;
+
+const VideoWrapper = styled.div`
+  position: relative;
+
+  overflow-y: visible;
+
+  ${breakpointsRange(
+    [{ prop: 'marginBottom', sizes: [60, 130], bases: [16, 20] }],
+    breakpoints.spacings
+  )};
+`;
+
+const Container = styled(SectionContainer)`
+  padding-bottom: 0;
+
+  ${breakpointsRange(
+    [
+      { prop: 'paddingTop', sizes: [98, 105] },
+      { prop: 'marginBottom', sizes: [202, 242] },
+    ],
+    breakpoints.spacings,
+    { bases: [16, 20] }
+  )};
+
+  ::before {
+    top: -60vh;
+
+    height: 60vh;
+  }
+
+  ::after {
+    bottom: -40vh;
+
+    height: 40vh;
+  }
+`;
+
+const TextContent = styled.div`
+  color: ${colors.gris100};
+
+  h2 {
+    ${breakpointsRange(
+      [
+        { prop: 'marginTop', sizes: [60, 90], bases: [16, 20] },
+        { prop: 'marginBottom', sizes: [16, 32], bases: [16, 20] },
+      ],
+      breakpoints.spacings
+    )};
+  }
+
+  p {
+    ${breakpointsRange(
+      [
+        { prop: 'fontSize', sizes: [16, 20], bases: [16, 20] },
+        { prop: 'marginBottom', sizes: [16, 32], bases: [16, 20] },
+      ],
+      breakpoints.spacings
+    )};
+  }
+`;
+
+const LimitDate = styled.p`
+  color: ${colors.bleu};
+  font-weight: ${fontWeights.bold};
+  font-family: ${fontFamilies.redaction};
+
+  ${breakpointsRange(
+    [
+      { prop: 'fontSize', sizes: [24, 40], bases: [16, 20] },
+      { prop: 'marginTop', sizes: [60, 90], bases: [16, 20] },
+      { prop: 'marginBottom', sizes: [60, 90], bases: [16, 20] },
+    ],
+    breakpoints.spacings
+  )};
+`;
+
+const SpeakersCall = () => {
+  const video = {
```

Comment:
Tu pourrais sortir la constante du component ici :
```suggestion
  const video = {
    ...
  }
  
  const SpeakersCall = () =>  (
    <>
      <SEO title='Appel à conférences' description='' />
      ....
  )
```